platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -146,9 +150,17 @@ public void createNextExperiment() {
             return;
         }
         try {
-            // if there is no benchmark running (and the queue has been
-            // initialized)
-            if ((experimentStatus == null) && (controller.queue != null)) {
+            // if there is no benchmark running, the queue has been
+            // initialized and cluster is healthy
+            ClusterManager clusterManager = new ClusterManagerImpl();
```

Comment:
The platform should have exactly one single instance of the manager. It might be created by the `PlatformController` given to the `ExperimentManager`. There is no need to create it again and again :wink: